hieradata/staging.yaml

```diff
@@ -564,3 +564,5 @@ users::pentest_machines:
 
 mongodb::s3backup::backup::s3_bucket: 'govuk-mongodb-backup-s3-staging'
 mongodb::s3backup::backup::s3_bucket_daily: 'govuk-mongodb-backup-s3-daily-staging'
+
+trade_tariff_uri: 'https://tariff-frontend-staging.cloudapps.digital/'
```

Comment:
I'm unsure as I don't have much Puppet experience. My only comment is that this bit of information may be used by other classes, and not just when configuring Plek, so it may be useful to not use that as the namespace (and perhaps use something like `service::` instead).